src/main/resources/templates/fragments/usersFragment.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en"
+      xmlns:th="http://www.thymeleaf.org"
+      xmlns:layout="http://www.ultraq.net.nz/thymeleaf/layout"
+      layout:decorate="~{layout/main}">
+<body>
+<!-- in usersFragment.html -->
+
+<div th:fragment="combinedFragment" id="replace-me" hx-swap="outerHTML">
+    <div class="flex flex-col items-center justify-center h-screen gap-10">
+
+        <div class="join">
+            <div>
```

Comment:
Why the bare div? A div is typically to add some styling, but if you're not doing anything with it, doesn't make sense to keep it.

Reply:
Yes! Thanks. Must be left over from something.